subsys/net/ip/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ zephyr_library_sources_ifdef(CONFIG_NET_STATISTICS   net_stats.c)
 zephyr_library_sources_ifdef(CONFIG_NET_TCP          connection.c tcp.c)
```

Comment:
The commit message of "net: core: Handle packets when packet sockets are enabled " is difficult to read.
What about writing it like this:
`If CONFIG_NET_SOCKETS_PACKET is enabled, then feed the packet to net_packet_socket_input() for processing. It will search the net_contexts and if proper handler is found, pass the packet to connection handler.`

Reply:
ok